### PR TITLE
Rename "BUILD_GENERAL" -> "BUILD"

### PR DESCRIPTION
**Pull request by digisomni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +41,7 @@ We encourage you to compile the documentation locally on your computer prior to
     C:\> choco install python
     ```
 
-4. Restart cmd or refresh with the command:
+4. Restart cmd as administrator or refresh with the command:
```
Comment (JulianGro) [inline README.md:44]: Have you confirmed this? On Linux you do not need elevated privileges for installing pip packages.

**Review by JulianGro — CHANGES_REQUESTED** [review]
Can you delete the old BUILD_GENERAL.md file as well?
Otherwise it is going to throw a warning every time.
```
2021-02-01T06:34:19.9153453Z /home/runner/work/vircadia-docs-sphinx/vircadia-docs-sphinx/docs/source/developer/build/BUILD_GENERAL.md: WARNING: document isn't included in any toctree
```

File: README.md (hunk 1 above)
Comment (digisomni) [reply README.md]: Yuppers, but only on Windows. It gives an error that doesn't make it clear that it's an administrator issue. I've updated it to be more clear.

Comment (JulianGro) [reply README.md]: These instructions are for Windows anyways, so info that this only needs administrative privileges on Windows seems unnecessary.
I just wanted to make sure that you confirmed it actually needed them.

Comment (digisomni) [reply README.md]: Ah, I might as well revise it to be clear in general lol since I assume the same commands past PowerShell work the same on Linux?

**Review by JulianGro — APPROVED** [review]
Lets just merge it and I will add Linux instructions with an extra PR

File: README.md (hunk 1 above)
Comment (JulianGro) [reply README.md]: It depends on the Linux distribution, but generally no. Also some Linux distributions will need additional packages.

**Review by JulianGro — APPROVED** [review]

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**